pkg/cmd/cli/cmd/env.go

```diff
@@ -234,7 +243,14 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 	}
 
 	skipped := 0
-	for _, info := range infos {
+	oldData := make([][]byte, len(infos))
+	for i, info := range infos {
+		old, err := json.Marshal(info.Object)
```

Comment:
Sorry, I still haven't pushed my latest changes. Latest doesn't do that ... most probably..